reference/build_helpers/cmake.rst

```diff
@@ -292,3 +292,38 @@ Environment variables
 
 There are some environment variables that will also affect the ``CMake()`` helper class. Check them in the
 :ref:`CMAKE RELATED VARIABLES<cmake_related_variables>` section.
+
+Example
+-------
+The following example of ``conanfile.py`` shows you how to manage a project with conan and CMake.
+
+.. code-block:: python
+
+    from conans import ConanFile, CMake
+
+    class SomePackage(ConanFile):
+        name = "SomePackage"
+        version = "1.0.0"
+        settings = "os", "compiler", "build_type", "arch"
+        generators = "cmake"
+
+    def configure_cmake(self):
+        cmake = CMake(self)
+
+        # put definitions here so that they are re-used in cmake between
+        # build(), test() and package()
+        cmake.definitions["SOME_DEFINITION_NAME"] = "On"
+
+        return cmake
+
+    def build(self):
+        cmake = self.configure_cmake()
+        cmake.build()
+
+    def test(self):
+        cmake = self.configure_cmake()
+        self.run("make check")
```

Comment:
The ``make check`` is too Makefile specific. I think the general approach should be ``cmake.test()``. If for some reason it is not working, please report.

Reply:
See the comment re this change above.